platforms/corda-ent/charts/float/files/firewall.conf

```diff
@@ -0,0 +1,37 @@
+firewallMode: FloatOuter
+
+# Uses /opt/corda/workspace as base directory
+networkParametersPath: "network-parameters"
+
+inboundConfig: {
+  listeningAddress: "{{ .Values.dmz.external}}:{{ .Values.node.p2pPort}}"
+}
+
+floatOuterConfig: {
+    floatAddress: "{{ .Values.dmz.internal}}:{{ .Values.bridge.tunnelPort }}"
+    expectedCertificateSubject: "{{ .Values.bridge.legalName }}"
+    tunnelSSLConfiguration: {
+           keyStorePassword: "password"
+           trustStorePassword: "password"
+           sslKeystore: "certificates/float.jks"
+           trustStoreFile: "certificates/trust.jks"
+    }
+}
+
+revocationConfig: { mode: "EXTERNAL_SOURCE"})
```

Comment:
check if ending ) is needed

Reply:
Can be removed. Fixed in [356b94a](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/893/commits/356b94a6b9bda311911f1dc595f0acb84792baa8)